service/src/test/java/bio/terra/workspace/common/utils/MockMvcUtils.java

```diff
@@ -1358,6 +1361,7 @@ public void assertClonedResourceMetadata(
             .sourceWorkspaceId(sourceWorkspaceId)
             .sourceResourceId(sourceResourceId));
 
+    // TODO (PF-2261): assert createdBy, lastUpdatedBy as well.
```

Comment:
Can the TODO also include asserting that createdDate and lastUpdatedDate aren't null?